app/src/main/java/com/framgia/soundclound/screen/home/HomeFragment.java

```diff
@@ -0,0 +1,41 @@
+package com.framgia.soundclound.screen.home;
+
+
+import android.databinding.DataBindingUtil;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
```

Comment:
ERROR: Unused import: android.support.v7.widget.RecyclerView. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports